mingw-w64-rustup/PKGBUILD

```
@@ -0,0 +1,79 @@
+_realname=rustup
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.28.1
+pkgrel=1
+pkgdesc="The Rust toolchain installer (mingw-w64)"
+arch=('x86_64')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url="https://github.com/rust-lang/rustup"
+license=('spdx:Apache-2.0 AND MIT')
+msys2_references=(
+	'archlinux: rustup'
+)
+depends=(
+	"${MINGW_PACKAGE_PREFIX}-curl"
+	"${MINGW_PACKAGE_PREFIX}-xz"
+	"${MINGW_PACKAGE_PREFIX}-zstd"
+)
+makedepends=(
+	"${MINGW_PACKAGE_PREFIX}-rust"
+	"${MINGW_PACKAGE_PREFIX}-cmake"
+	"${MINGW_PACKAGE_PREFIX}-nasm"
```

by the way, aws-lc-rs also use rust bindgen
```suggestion
	"${MINGW_PACKAGE_PREFIX}-nasm"
	"${MINGW_PACKAGE_PREFIX}-rust-bindgen"
```